js/id/ui/raw_tag_editor.js

```diff
@@ -186,7 +186,7 @@ iD.ui.RawTagEditor = function(context) {
             .on('change', keyChange);
 
         $items.select('input.value')
-            .value(function(d) { return d.value; })
+            .attr('value', function(d) { return d.value; })
```

Comment:
If I kept both lines of code (and changed the name of the attr), would it be too bloated? I couldn't get cucumber to check the value unless it was a css selector.